e2e/johtoselvitys_tilaus_taydennyspyynto.spec.ts

```diff
@@ -0,0 +1,176 @@
+import { test, expect } from '@playwright/test';
+import { perustaja, vastaava, suorittaja, rakennuttaja, asianhoitaja, testiData, testiOsoite, tarkistaTulokset } from './_setup';
```

Comment:
SonarCloudin mukaan "rakennuttajan" import on turha tässä.
https://sonarcloud.io/project/issues?sinceLeakPeriod=true&issueStatuses=OPEN%2CCONFIRMED&pullRequest=1029&id=City-of-Helsinki_haitaton-ui&open=AZQsaLYvp3pLVeoorkB4

Reply:
Ja samoin on "asianhoitaja" turha
https://sonarcloud.io/project/issues?sinceLeakPeriod=true&issueStatuses=OPEN%2CCONFIRMED&pullRequest=1029&id=City-of-Helsinki_haitaton-ui&open=AZQsaLYvp3pLVeoorkB5